src/app/(general)/auth/signin/page.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from 'next/image'
+
+import backgroundImage from '@public/images/background-graphic.png'
+import OAuthButtons from '@/components/sections/auth/OAuthButtons'
+
+const SignInPage = () => {
+  return (
+    <main className="relative flex flex-col justify-between pt-[200px]">
+      <Image
+        src={backgroundImage}
+        alt="Background image"
+        width={514}
+        className="opacity-10"
+        style={{
+          position: 'absolute',
+          right: 0,
+          top: -136,
+          zIndex: -1,
+        }}
+      />
+      <div className="flex flex-col items-center justify-center px-24">
+        <h1 className="whitespace-pre-line text-headlineL font-bold uppercase">
+          LOGIN
+        </h1>
+        <p className="pt-2">
+          Define the reward for the project (in ADA), severity rewards, known
+          issues and audit period.
+        </p>
+        <div className="pt-10">
```

Comment:
Please check the correct spacing for components (it's visible when you hold `alt` key) - should be `mt-3` for text and `mt-12` for buttons. Also, there's 155px bottom padding below the buttons and footer and the subtext was updated as well.

Reply:
I didn't know about that `alt` key in figma. Thanks for pointing this out!